doc/gallery/gallery.rst

```diff
@@ -124,12 +129,30 @@ Research papers
 
 Finally, some links to studies and research papers that utilize Strawberry Fields.
 
-#. D\. Su, K. K. Sabapathy, C. R. Myers, H. Qi, C. Weedbrook, and K. Brádler. *Implementing quantum algorithms on temporal photonic cluster states.* arXiv, 2018. `arXiv:1805.02645 <https://arxiv.org/abs/1805.02645>`_.
+#. D\. Su, K. K. Sabapathy, C. R. Myers, H. Qi, C. Weedbrook, and K. Brádler. *Implementing quantum algorithms on temporal photonic cluster states.* `Physical Review A 98, 032316 <https://journals.aps.org/pra/abstract/10.1103/PhysRevA.98.032316>`_, 2018.
```

Comment:
These could be included in `references.bib` and listed automatically using sphinxcontrib-bibtex.